src/components/com_tjcertificate/plugins/tjqueue/certificate/certificate.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * @package    Techjoomla.Libraries
+ * @author     Techjoomla <extensions@techjoomla.com>
+ * @copyright  Copyright (c) 2009-2021 TechJoomla. All rights reserved.
+ * @license    GNU General Public License version 2 or later.
+ */
+
+// No direct access
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Plugin\CMSPlugin;
+
+/**
+ * TjQueue
+ *
+ * @package     Techjoomla.Libraries
+ * @subpackage  Tjqueue
+ * @since       1.0
+ */
+class PlgTjqueueRecords extends CMSPlugin
```

Comment:
Plugin name and plugin class name is different?

Reply:
this type of structure needed in cli, check following ref code of cli

$filePath = JPATH_SITE . '/plugins/tjqueue/' . $plugin . '/consumers/' . $class . '.php';